packages/web/src/new/sdk/Analytics/hooks/useTrackOpenPageAction.ts

```diff
@@ -5,12 +5,12 @@ import { trackEvent } from '../amplitude';
 import type { OpenPageActionNames } from '../types';
 
 export const useTrackOpenPageAction = (eventName: OpenPageActionNames) => {
-  const location = useLocation<{ fromPage?: string }>();
+  const location = useLocation<{ redirectTo?: string }>();
```

Comment:
in this case (when change property name) we need to rename it all over the code. IMHO: it's better to leave old name. Because redirectTo is sutable only in this current case. In other contexts we can learn user's route's path by fromPage